spec/models/photo_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Photo, type: :model do
+  it { is_expected.to belong_to(:user)}
+
+  feature '#username' do
+    it 'returns the username of the owner' do
+      user = User.create(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)